src/routing/bootstrap.rs

```diff
@@ -167,6 +170,10 @@ impl<'a> State<'a> {
                     );
                     bootstrap_addrs = new_bootstrap_addrs.to_vec();
                 }
+                GetSectionResponse::SectionInfrastructureError(error) => {
+                    error!("Handle infrastructure error: {:?}", error);
+                    // TODO: handle
```

Comment:
nor here

Reply:
as a bootstrapping node, you shall not receive such SectionInfrastructureError ?
as above, you can just log it.